Resources/Prototypes/_CD/Loadouts/Jobs/Science/titles.yml

```
@@ -0,0 +1,8 @@
+﻿- type: loadout
+  id: RoboticistPDA
+  equipment: RoboticistPDA
```

```suggestion
  equipment:
    id: RoboticistPDA
```
In this case, the error message was actually kinda intelligible because it was saying "I don't know how to turn just `RoboticistPDA` into a `System.Collections.Generic.Dictionary`2[System.String,Robust.Shared.Prototypes.EntProtoId]`", which unmangles to `Dictionary<string, EntProtoId>` which, yeah, that usually is a key-value object, not just a string.